Index: configure.ac
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/configure.ac b/configure.ac
--- a/configure.ac	(revision Staged)
+++ b/configure.ac	(date 1745108464300)
@@ -2091,8 +2091,6 @@
   ])
 ], [[#include <libyang/libyang.h>]])

-AC_CHECK_LIB([yang],[lyd_find_xpath3],[],[AC_MSG_ERROR([m4_normalize([
-libyang missing lyd_find_xpath3])])])
 dnl -- don't add lyd_new_list3 to this list unless bug is fixed upstream
 dnl -- https://github.com/CESNET/libyang/issues/2149
 AC_CHECK_FUNCS([ly_strerrcode ly_strvecode lyd_trim_xpath])
